_episodes/03-benchmark-and-scaling.md

```diff
@@ -1,17 +1,20 @@
 ---
 title: "Benchmarking and Scaling"
-teaching: 0
-exercises: 0
+teaching: 15
+exercises: 15
 questions:
 - "What is benchmarking?"
-- "How do I benchmark an application?"
+- "How do I do a benchmark in LAMMPS?"
 - "What is scaling?"
 - "How do I perform a scaling analysis?"
 objectives:
-- "Be able to perform a benchmark analysis of an application"
-- "Be able to perform a scaling analysis of an application"
+- "Be able to perform a benchmark analysis of LAMMPS"
+- "Be able to perform a scaling analysis of LAMMPS"
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Benchmarking is a way of assessing the performance of a program or set of programs"
+- "The `log.lammps` file shows important information about the timing, processor layout, etc. 
+  which you can use to record your benchmark"
+- "Scaling is the effective use of computational resources as you increase the size of your workload"
```

Comment:
With strong scaling you wouldn't be increasing your workload. I think I would probably say
```
"Scaling" concerns the effective use of computational resources. Two types are typically discussed: strong scaling (where we increase the compute resources but keep the problem size the same) and weak scaling (where we increase the problem size in proportion to our increase of compute resources)
```